components/ShortcutsDialog.vue

```diff
@@ -0,0 +1,104 @@
+<template>
+  <v-dialog
+    v-model="dialog"
+    width="500"
+  >
+    <template v-slot:activator="{ on }">
+      <v-btn
+        x-small
+        outlined
+        v-on="on"
+      >
+        ⌨️ Shortcuts
+      </v-btn>
+    </template>
+
+    <v-card>
+      <v-card-title
+        class="headline grey lighten-2"
+        primary-title
+      >
+        Available shortcuts
+      </v-card-title>
+
+      <v-card-text>
+        <v-list-item>
+          <v-list-item-content>
+            <v-list-item-title>
+              <v-chip
```

Comment:
This doesn't need 6 lines.  Splitting it up that way makes it _harder_ to read.

```
<v-chip
 label
>
  Space
</v-chip>
– Play/Pause
```

vs

```
<v-chip label>Space</v-chip> – Play/Pause
```